.travis.yml

```diff
@@ -4,5 +4,9 @@ python:
 
 # command to install dependencies
 install: pip install -r requirements.txt
+# enforce some code style standards
+before_script: 
+  - pip install flake8 --use-mirrors
+  - flake8 --ignore=E731,E226 .
```

Comment:
@mrshu I never thought you, of all people, would be a people-pleaser :-D

Anyway, okay I agree that they might not always be in the best interest.